src/win/util.c

```diff
@@ -207,13 +207,102 @@ int uv_cwd(char* buffer, size_t* size) {
 }
 
 
+static int uv__is_win32_namespace(const WCHAR* path, size_t len) {
+  return len >= 4 &&
+         path[0] == L'\\' &&
+         path[1] == L'\\' &&
+         (path[2] == L'?' || path[2] == L'.') &&
+         path[3] == L'\\';
+}
+
+
+static int uv__is_nt_namespace(const WCHAR* path, size_t len) {
+  return len >= 4 &&
+         path[0] == L'\\' &&
+         path[1] == L'?' &&
+         path[2] == L'?' &&
+         path[3] == L'\\';
+}
+
+
+static int uv__is_unc_after_prefix(const WCHAR* path, size_t len) {
+  if (len < 4)
+    return 0;
+  
+  return _wcsnicmp(path, L"UNC", 3) == 0 && path[3] == L'\\';
```

Comment:
```suggestion
  return len >= 4 && _wcsnicmp(path, L"UNC\\", 4) == 0;
```
I'd implement uv__is_nt_namespace the same way.